src/main/views/home.njk

```diff
@@ -1,13 +1,37 @@
 {% extends "template.njk" %}
-
-{% block pageTitle %}
-  Home
-{% endblock %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
 
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <h1 class="govuk-heading-xl">Home Page</h1>
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-4 ">{{ pageTitle }}</h1>
+      <p class="govuk-body">{{ serviceInfo }}</p>
+      {{ govukInsetText({
+        text: infoTxt
+      }) }}
+      {{ govukWarningText({
+        html: warningTxt1,
+        iconFallbackText: warningTxt
+      }) }}
+      <p class="govuk-body ">{{ saveInfo }}</p>
+      {{ govukWarningText({
+        html: warningTxt2,
+        iconFallbackText: warningTxt
+      }) }}
+      {{ govukButton({
+        text: "Start now",
```

Comment:
One small thing, this text should be coming from translation file.

Reply:
Change applied - build successful